lib/enthrift/server/TNonblockingServer.py

```diff
@@ -18,25 +18,30 @@
 #
```

Comment:
Similar question as re: TJSONProtocol - should we include "server" oriented types in our client SDK?